tests/test_events.py

```diff
@@ -29,42 +28,45 @@ async def _ignore_test_consoleLog_logEntryAddedEventEmitted(websocket):
             "expression": "console.log('some log message')",
             "target": {"context": context_id}}})
 
+    # The order of the two responses are not guarenteed
+    resp1 = await read_JSON_message(websocket)
+    resp2 = await read_JSON_message(websocket)
+    if "id" in resp1:
+        resp1, resp2 = resp2, resp1
+
     # Assert "log.entryAdded" event emitted.
-    resp = await read_JSON_message(websocket)
     recursiveCompare(
-        resp,
+        resp1,
         {
             "method": "log.entryAdded",
             "params": {
                 # BaseLogEntry
                 "level": "info",
-                "text": "some log message",
                 "timestamp": "__any_value__",
-                "stackTrace": [{
-                    "url": "__any_value__",
-                    "functionName": "",
-                    "lineNumber": 0,
-                    "columnNumber": 8}],
+                "stackTrace": {
+                    "callFrames": [{
+                        "url": "__any_value__",
```

Comment:
Don't we know for sure the `url` at this time?

Reply:
updated url value in the test.